plugins/woocommerce/assets/js/admin/meta-boxes-product-variation.js

```diff
@@ -807,6 +807,10 @@ jQuery( function( $ ) {
 				default :
 					$( 'select.variation_actions' ).trigger( do_variation_action );
 					data = $( 'select.variation_actions' ).triggerHandler( do_variation_action + '_ajax_data', data );
+					
+					if ( true === data.cancel ) {
```

Comment:
Let's change this to a `null` check on `data` instead.

Reply:
I changed this as per your request. I noticed that all the assets were moved to the legacy subfolder. Would I need to change the file in the new path instead?